tatus

```diff
@@ -0,0 +1,22 @@
+[1mdiff --git a/azure-mgmt-sql/src/main/java/com/microsoft/azure/management/sql/SqlFirewallRule.java b/azure-mgmt-sql/src/main/java/com/microsoft/azure/management/sql/SqlFirewallRule.java[m
```

Comment:
Looks like I pressed some shortcut on console and a file is created with diff of changes, removing it